CHANGELOG.md

```diff
@@ -2,6 +2,15 @@
 
 ## Unreleased
 
+### Breaking Changes
+
+- `Environment` and `Dist` get/set functions were removed from the `Scope` class and now these properties have to be set in plugin settings instead.
+- `ConfigureScope` function was removed from `SentrySubsystem` class.
```

Comment:
this is probably ok without deprecating first because it hasn't reached 1.0.0?

Reply:
Yes, given the scope of the changes in this PR it seems like an appropriate time to remove these APIs ahead of the major version bump.